bag_tracker/src/main/scala/uk/ac/wellcome/platform/archive/bag_tracker/BagTrackerApi.scala

```diff
@@ -0,0 +1,50 @@
+package uk.ac.wellcome.platform.archive.bag_tracker
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+import akka.stream.Materializer
+import grizzled.slf4j.Logging
+import uk.ac.wellcome.platform.archive.common.storage.models.StorageManifest
+import uk.ac.wellcome.typesafe.Runnable
+
+import scala.concurrent.ExecutionContextExecutor
+
+class BagTrackerApi(host: String, port: Int)(
+  implicit
+  actorSystem: ActorSystem,
+  materializer: Materializer
+) extends Runnable
+    with Logging {
+  implicit val ec: ExecutionContextExecutor = actorSystem.dispatcher
+
+  val route: Route =
+    pathPrefix("bags") {
+      concat(
+        post {
+          entity(as[StorageManifest]) { manifest =>
+            println(manifest)
+            println("create the manifest!")
+            complete(StatusCodes.Created)
+          }
+        },
+        pathSuffix("versions" /) {
+          path(Segment / Remaining) { (space, remaining) =>
+            println(s"space = $space")
```

Comment:
does this need a GET?

Reply:
Yes, it does. Oops. Coming in the next pull request, so I'm not going to fix it here.